modules/artic/minion/main.nf

```diff
@@ -43,7 +43,9 @@ process ARTIC_MINION {
         summary = ""
         model = file(medaka_model).exists() ? "--medaka-model ./$medaka_model" : "--medaka-model $medaka_model"
     }
+    def hd5_plugin_path = params.options.hd5_plugin_path ? "HDF5_PLUGIN_PATH=" + params.options.hd5_plugin_path : "HDF5_PLUGIN_PATH=/usr/local/lib/python3.6/site-packages/ont_fast5_api/vbz_plugin"
     """
+    $hd5_plugin_path \\
```

Comment:
```suggestion
    def hd5_plugin_path = task.ext.hd5_plugin_path ? "HDF5_PLUGIN_PATH=" + task.ext.hd5_plugin_path : "HDF5_PLUGIN_PATH=/usr/local/lib/python3.6/site-packages/ont_fast5_api/vbz_plugin"
    """
    $hd5_plugin_path

```

Reply:
Ok. I think this is the best workaround if we need to provide the path manually (and in line with the current DSL2 syntax we are using).

Reply:
Once this is merged I will re-install it in viralrecon @aunderwo. Hoping to get a release out soon anyway.